package.json

```diff
@@ -19,12 +19,14 @@
     "serve": "docusaurus serve",
     "format": "prettier --write .",
     "format:check": "prettier --check .",
+    "prettier": "prettier",
```

Comment:
❓ Did you mean to add this?

Reply:
I did! I don't really understand what's going on but for some reason running prettier to format the API pages right after they're generated ([example](https://github.com/camunda/camunda-docs/blob/10b826ea71aad1bd8c2fac68227b14d4b3d6b579/package.json#L29)) doesn't actually work, and I keep having to format things manually myself afterward! Adding this line allows me to do something like `npm run prettier -- --write api/zeebe/docs/` in my terminal, thus preventing me from accidentally committing a hardcoded path on line 20 above it, like I did in #3435.

Reply:
You saw where I was coming from here 😄